templates/method.mustache

```diff
@@ -17,21 +17,33 @@
             {{^isSingleton}}
                 {{#isPatternType}}
                     Object.keys(parameters).forEach(function(parameterName) {
-                    if(new RegExp('{{&pattern}}').test(parameterName)){
-                    queryParameters[parameterName] = parameters[parameterName];
-                    }
+                        if(new RegExp('{{&pattern}}').test(parameterName)){
+                            queryParameters[parameterName] = this.convertParameterCollectionFormat(
+                                parameters[parameterName],
+                                '{{&collectionFormat}}'
+                            );
+                        }
                     });
                 {{/isPatternType}}
                 {{^isPatternType}}
-                    if(parameters['{{&camelCaseName}}'] !== undefined){
-                    queryParameters['{{&name}}'] = parameters['{{&camelCaseName}}'];
+                    if(parameters['{{&camelCaseName}}'] !== undefined) {
+                        queryParameters['{{&name}}'] = this.convertParameterCollectionFormat(
+                            parameters['{{&camelCaseName}}'],
+                            '{{&collectionFormat}}'
+                        );
                     }
                 {{/isPatternType}}
             {{/isSingleton}}
         {{/isQueryParameter}}
 
         {{#isPathParameter}}
-            path = path.replace('{{=<% %>=}}{<%&name%>}<%={{ }}=%>', `${encodeURIComponent(parameters['{{&camelCaseName}}'].toString())}`);
+            path = path.replace(
+                '{{=<% %>=}}{<%&name%>}<%={{ }}=%>',
```

Comment:
Is it possible to avoid changing the escape characters for this parameter usage?

Reply:
(same below)

Reply:
I do not understand what is meant. Can you make a suggestion here or implement it yourself?

Reply:
`{{=<% %>=}}` changes the default escaping characters to `<%` and `%>` instead of `{{` and `}}`
However, I see that you didnt touch that part, so you can ignore my comment (and I can clean that up later :) )